website/server/controllers/api-v3/hall.js

```diff
@@ -175,7 +175,7 @@ api.getHero = {
     if (validator.isUUID(heroId)) {
       query = {_id: heroId};
```

Comment:
@Alys do you think I should apply the case-insensitive check to the UUID search as well? If all UUIDs are stored lowercase, it may be beneficial in case an uppercase letter makes it to this point.